src/fft.js

```diff
@@ -171,13 +171,12 @@ class FFT {
    *
    */
   waveform() {
-    var bins, mode;
+    var mode;
     var normalArray = new Array();
 
     for (var i = 0; i < arguments.length; i++) {
       if (typeof arguments[i] === 'number') {
-        bins = arguments[i];
-        this.analyser.fftSize = bins * 2;
```

Comment:
nice improvement here, since `this.bins` has a setter that also updates `this.analyser.fftSize` 💯 
https://github.com/processing/p5.js-sound/blob/17987e714cf5bce6904e31912603a0d640e8d0c9/src/fft.js#L92-L98